config/schema/base_document_type.json

```diff
@@ -0,0 +1,22 @@
+{
+  "fields": [
+    "format",
+    "title",
+    "title",
```

Comment:
@dsingleton pointed out to me that there are dupes in this list - there aren't meant to be - will fix in a bit.